cabot/templates/cabotapp/_statuscheck_list.html

```diff
@@ -15,9 +15,6 @@
       {% if checks_type == "All" or checks_type == "Jenkins" %}
       &nbsp;<a href="{% url "create-jenkins-check" %}?instance={{ instance.id }}&service={{ service.id }}" class="" title="Add new Jenkins check"><i class="glyphicon glyphicon-plus"></i><i class="glyphicon glyphicon-ok"></i></a>
```

Comment:
Should get rid of "instance" in this url also (might be good to check the other templates/grep the templates directory for cases like this)